python/cudf/cudf/tests/test_indexing.py

```diff
@@ -1305,28 +1305,33 @@ def test_loc_datetime_index(sli, is_dataframe):
             }
         ),
         cudf.DataFrame(index=[1, 2, 3]),
-        cudf.DataFrame(index=range(1000000)),
+        cudf.DataFrame(index=range(10000)),
```

Comment:
I assume the size changes are unrelated to the fix, just to speed up the tests. Do they even need to be as big as they are? Would a few 10s or 100s of rows be enough?